demisto_sdk/commands/content_graph/strict_objects/script.py

```diff
@@ -0,0 +1,96 @@
+from typing import List, Optional
+
+from pydantic import Field
+
+from demisto_sdk.commands.common.constants import (
+    SKIP_PREPARE_SCRIPT_NAME,
+    TYPE_PYTHON2,
+    TYPE_PYTHON3,
+    MarketplaceVersions,
+)
+from demisto_sdk.commands.common.StrEnum import StrEnum
+from demisto_sdk.commands.content_graph.strict_objects.base_strict_model import (
+    Argument,
+    BaseIntegrationScript,
+    BaseStrictModel,
+    CommonFields,
+    Important,
+    Output,
+    ScriptType,
+    create_dynamic_model,
+)
+from demisto_sdk.commands.content_graph.strict_objects.common import create_model
+
+COMMENT_DYNAMIC_MODEL = create_dynamic_model(
+    field_name="comment",
+    type_=Optional[str],
+    default=None,
+    suffixes=[MarketplaceVersions.MarketplaceV2.value],
+    include_without_suffix=True,
+)
+
+
+class SkipPrepare(StrEnum):
+    SCRIPT_NAME = SKIP_PREPARE_SCRIPT_NAME
+
+
+class ScriptSubType(StrEnum):
+    PYTHON3 = TYPE_PYTHON3
+    PYTHON2 = TYPE_PYTHON2
+
+
+class CommonFieldsScript(CommonFields):  # type:ignore
+    id_x2: Optional[str] = None
+    id_xpanse: Optional[str] = Field(None, alias="id:xpanse")
```

Comment:
let's generate a dynamic model instead

Reply:
since it's a type of a field in the StrictScript object (common_fields: CommonFieldsScript), it does not work for me to make it dynamic model